### PR TITLE
Don't send unsolicited emails either

### PR DESCRIPTION
**Pull request by bubbafat**
I've been getting spam from recruiters after posting in #jobs - I'd like to have the ability to refer them back to the channel policy directing them to not do that.

### CODE REVIEW

File: channel-policies.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Here you can find postings for development jobs in the Triangle area
 
 Rules:
 
-1. Do not send unsolicited DMs to members about job openings
+1. Do not send unsolicited DMs or emails to members about job openings
```
Comment (nilbus) [inline channel-policies.md:18]:
```suggestion
1. Do not send unsolicited DMs or emails to members about job openings or other commercial solicitations
```

Might this be better to also cover these other spam messages you received?
> So far I have gotten spam for:
> - “We have jobs, come apply”
> - “We have contractors, come hire them.”
> - “We have Product XXX, come buy it.”

Are there any legitimate solicitations by DM we'd unintentionally ruling against?

Comment (bubbafat) [reply channel-policies.md:18]: Maybe adding "commercial solicitations"?  

I could see something like "Hey, you just posted a Scala job opening and I run the Triangle Scala Whatever Group.  I just wanted to let you know that we have a monthly meetup of Scala engineers and at the end of each meeting we invite hiring companies to share their relevant job openings."

Something like that would be welcomed (by me, anyway).

Comment (nilbus) [reply channel-policies.md:18]: 👍 Updated.

Comment (bbbco) [inline channel-policies.md:18]: What about using the term "communications" in order to make this a generic enough policy to cover other types of communication methods?

`Do not send unsolicited communications (DMs, emails, calls, etc) to members about job openings or other commercial solicitations`

**Comment by BrandonMathis** [comment]
Ty for these updates. Any additional modifications can always be proposed via a new PR 👍🏼

**Pull request merged**